sklearn/svm/base.py

```diff
@@ -779,6 +779,15 @@ def _fit_liblinear(X, y, C, fit_intercept, intercept_scaling, class_weight,
     if verbose:
         print('[LibLinear]', end='')
 
+    # LinearSVC breaks when intercept_scaling is <= 0
+    if fit_intercept is True:
+        if intercept_scaling is None:
```

Comment:
I was getting an Unorderable types error as `intercept_scaling <= 0` was comparing None to an int when that test in test_bounds.py passed None to intercept_scaling.